docs/how-to/h-deploy-aws.md

```diff
@@ -0,0 +1,193 @@
+
+To check they are all correctly installed, you can run the commands demonstrated below with sample outputs:
+
+```console
+~$ juju version
+3.5.4-genericlinux-amd64
+
+~$ aws --version
+aws-cli/2.13.25 Python/3.11.5 Linux/6.2.0-33-generic exe/x86_64.ubuntu.23 prompt/off
+```
+### Authenticate
+[Create an IAM account](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html) (or use legacy access keys) to operate AWS EC2:
+```shell
+mkdir -p ~/.aws && cat <<- EOF >  ~/.aws/credentials.yaml
+credentials:
+  aws:
+    NAME_OF_YOUR_CREDENTIAL:
+      auth-type: access-key
+      access-key: SECRET_ACCESS_KEY_ID
+      secret-key: SECRET_ACCESS_KEY_VALUE
+EOF
+```
+
+<!--- TODO, teach Juju to use `aws configure` format:
+```shell
+~$ aws configure
+AWS Access Key ID [None]: SECRET_ACCESS_KEY_ID
+AWS Secret Access Key [None]: SECRET_ACCESS_KEY_VALUE
+Default region name [None]: eu-west-3
+Default output format [None]:
+```
+Check AWS credentials:
+```shell
+~$ aws sts get-caller-identity
+{
+    "UserId": "1234567890",
+    "Account": "1234567890",
+    "Arn": "arn:aws:iam::1234567890:root"
+}
+```
+--->
+
+## Bootstrap Juju controller on AWS EC2
+
+Add AWS credentials to Juju:
+```shell
+juju add-credential aws -f ~/.aws/credentials.yaml
+```
+Bootstrap Juju controller ([check all supported configuration options](https://juju.is/docs/juju/amazon-ec2)):
+```shell
+juju bootstrap aws <CONTROLLER_NAME>
+```
+[details="Output example"]
+```shell
+> juju bootstrap aws
+Creating Juju controller "aws-us-east-1" on aws/us-east-1
+Looking for packaged Juju agent version 3.5.4 for amd64
+Located Juju agent version 3.5.4-ubuntu-amd64 at https://juju-dist-aws.s3.amazonaws.com/agents/agent/3.5.4/juju-3.5.4-linux-amd64.tgz
+Launching controller instance(s) on aws/us-east-1...
+ - i-0f4615983d113166d (arch=amd64 mem=8G cores=2)           
+Installing Juju agent on bootstrap instance
+Waiting for address
+Attempting to connect to 54.226.221.6:22
+Attempting to connect to 172.31.20.34:22
+Connected to 54.226.221.6
+Running machine configuration script...
+Bootstrap agent now started
+Contacting Juju controller at 54.226.221.6 to verify accessibility...
+
+Bootstrap complete, controller "aws-us-east-1" is now available
+Controller machines are in the "controller" model
+
+Now you can run
+	juju add-model <model-name>
+to create a new model to deploy workloads.
+```
+[/details]
+
+You can check the [AWS EC2 instance availability](https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#Instances:instanceState=running) (ensure the right AWS region chosen!):
+![image|690x118](upload://putAO5NyHdaeWE6jXI8X1hZHTYv.png)
+
+## Deploy charms
+
+Create a new Juju model, if needed:
+```shell
+juju add-model <MODEL_NAME>
+```
+> (Optional) Increase the debug level if you are troubleshooting charms:
+> ```shell
+> juju model-config logging-config='<root>=INFO;unit=DEBUG'
+> ```
```

Comment:
The idea would be to provide a box in the documentation, kinda used for things like warnings, disclaimers, etc, things you would like to box and to highlight. 

Sometimes there are cases where this can be nice to have. Maybe worth discussing with @a-velasco if we - as DataPlatform - should advocate against it or in which cases we should use this

Reply:
OK, that's called admonition. Sometimes we use quotation to do a simple admonition, for example:
```
> **NOTE**: This is a noteworthy admonition.
```
But ideally, we should use a proper syntax, which should work better for more complicated admonitions with multiple elements in it, like code blocks. For example:
```
[note type="caution"]
Always clean AWS resources that are no longer necessary! Abandoned resources are tricky to detect and they can become expensive over time.
[/note]
```